code/modules/overmap/exoplanets/exoplanet.dm

```diff
@@ -424,6 +424,12 @@
 			total_moles = max(total_moles - part, 0)
 			i++
 
+/// Randomly generated planets should call parent and append to `ground_survey_result`.
+/// Lore planets should just set it to one static string.
```

Comment:
This says what to do with it, but not what this proc is for, document what this proc is for
Also, this needs to go where the proc is defined, not on a child implementation, otherwise you'll never see it

Reply:
done